jgrapht-ext/src/test/java/org/jgrapht/ext/DOTUtilsTest.java

```diff
@@ -73,7 +73,8 @@ public void testConvertGraphToDotString()
 
         //@formatter:off
         String expectedGraphAsDotString =
-                "digraph G {"            +
+                DOTExporter.DONT_ALLOW_MULTIPLE_EDGES_KEYWORD + " " +
+                  DOTExporter.DIRECTED_GRAPH_KEYWORD + " G {" +
                   "  1 [ label=\"A\" ];" +
                   "  2 [ label=\"B\" ];" +
                   "  3 [ label=\"C\" ];" +
```

Comment:
Since you updated the importer and the exporter, you need to add additional tests to make sure that your changes are working properly. 

Add at least 2 different tests in the `DOTImporter` when you read the id from a graph. One with valid and one with invalid id. 

You also need to add 2-3 tests for the case where the graph header is missing things, like `strict` or the ID, or both.